specification/vmwarevirtustream/resource-manager/Microsoft.VMwareVirtustream/preview/2016-09-01-preview/vmwarevirtustream.json

```diff
@@ -0,0 +1,1199 @@
+        ],
+        "responses": {
+          "200": {
+            "description": "successful operation",
+            "schema": {
+              "$ref": "#/definitions/GetAdminCredentialsResponse"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed",
+            "schema": {
+              "$ref": "#/definitions/ApiError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "PrivateClouds_GetAdminCredentials": {
+            "$ref": "./examples/PrivateClouds_GetAdminCredentials.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ApiError": {
+      "properties": {
+        "error": {
+          "$ref": "#/definitions/ApiErrorBase"
+        }
+      },
+      "description": "API error response"
+    },
+    "ApiErrorBase": {
+      "properties": {
+        "code": {
+          "type": "string",
+          "description": "Error code"
+        },
+        "message": {
+          "type": "string",
+          "description": "Error message"
+        }
+      },
+      "description": "Api error."
+    },
+    "OperationList": {
+      "description": "Pageable list of operations",
+      "properties": {
+        "value": {
+          "description": "List of operations",
+          "type": "array",
+          "readOnly": true,
+          "items": {
+            "$ref": "#/definitions/Operation"
+          }
+        },
+        "nextLink": {
+          "description": "URL to get the next page if any",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "Operation": {
+      "description": "A REST API operation",
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "Name of the operation being performed on this object",
+          "type": "string",
+          "readOnly": true
+        },
+        "display": {
+          "description": "Contains the localized display information for this operation",
+          "readOnly": true,
+          "properties": {
+            "provider": {
+              "description": "Localized friendly form of the resource provider name",
+              "type": "string",
+              "readOnly": true
+            },
+            "resource": {
+              "description": "Localized friendly form of the resource type related to this operation",
+              "type": "string",
+              "readOnly": true
+            },
+            "operation": {
+              "description": "Localized friendly name for the operation",
+              "type": "string",
+              "readOnly": true
+            },
+            "description": {
+              "description": "Localized friendly description for the operation",
+              "type": "string",
+              "readOnly": true
+            }
+          }
+        }
+      }
+    },
+    "ExpressRouteAuthorization": {
+      "properties": {
+        "id": {
```

Comment:
what id is this? how does the customer get it?

Reply:
It is a read-only property that you get back when an authorization is created. You pass in `name` and you get back an ìd` and a `key`.

Reply:
In that case, please could you mark this + key as read-only (`"readOnly": true`)

Reply:
Added.